src/assets/cs-guide.md

```diff
@@ -29,8 +29,84 @@ Please add yourself to this list (link is optional) if you've contributed to thi
 
 # Courses
 
+
+###### Requirements Legend
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Contributing](#contributing)
+- [Contributors](#contributors)
+
+# Introduction
+
+---
+
+This is an open-source guide for computer science students at Brock University.
+
+# Contributing
+
+---
+
+Everyone is _highly_ encouraged to contribute to this guide so that we can make this a very useful resource for students. Add any information you might find useful and feel free to add/change sections.
+
+Anyone can contribute to this document through GitHub's editor - no coding/git skills necessary. Click the "**Contribute to guide**" button at the top of this page and it will bring you to GitHub's text editor. You can make/preview changes on there and create a pull request when you're done. An admin will then approve your changes and add it to production. The document is written in [markdown](https://www.markdownguide.org/basic-syntax/) syntax.
+
+If you're posting any opinionated information, use your name in the sentence (i.e. John recommends...).
+
+## Contributors
+
+Please add yourself to this list (link is optional) if you've contributed to this guide.
+
+- [Dylan Souvage](https://github.com/firefelix)
+- [Kindeep Singh Kargil](https://kindeep.me)
+
+# Courses
+(Note: do not rely on this, always consult [BrockU Timetable](https://brocku.ca/guides-and-timetables/timetables/?session=fw&type=ug&level=all) and your [Course Calendar](https://brocku.ca/webcal/current/undergrad/cosc.html))
+
+###### Requirements Legend
+
+❤️ Bachelor of Science, Computer Science (Honours & General) Required Course
+
+💜 Computer Science (COSC) Elective
+
+💛 Applied Computing Minor (APCO) Required Course/APCO Elective
+
+
+| Course Code | Course Name | Fall | Winter | Spring | Required | Prerequisite(s)
+|-------------|:------------------------------------------------:|-------------|-------------|---------------------|----------|:------------------------------------------------------------------------------:|
+| APCO-1P00 | Introduction to Media Computation | ✅ | ✅ | ✅ | 💛| |
+| APCO-1P01 | Fluency with Technology | ✅ | ✅ | ✅ | 💛| | 
+| APCO-1P30 | Programming for Interactive Media | ❌ | ✅ | ❌ | 💛| |
+| COSC-1P50 | Integrity and Literacy in the Information Age | ✅ | ✅ | ✅ | 💛❤️| |
+| APCO-1P93 | Applied Programming |  ❌ | ✅ |  ❌ | 💛| APCO-1P00 or 4U Mathematics credit |
+| APCO-2P11 | Personal Computers and Networks |  ❌ | ✅ |  ❌ | 💛| APCO-1P01 |
+| COSC-2P89 | Internet Technologies |  ❌ | ✅ |  ❌ | 💛💜| APCO 1P00 (minimum 60 percent) and 1P01 (minimum 60 percent) or COSC 1P03 (minimum 60 percent) |
+| COSC-3P94 | Human Computer Interaction |  ❌ | ✅ |  ❌ | 💛💜| Two Credits from either IASC/APCO/COSC |
+
+
+## School Resources
```

Comment:
It looks like School Resources and goodies were repeated?

Reply:
Actually looks like all previous content was somehow repeated a second time

Reply:
whoops, might have to do with the way I edited it, can you resolve this from here or should I submit a new PR

Reply:
i think i fixed it